KSystemInformer/cid_tracking.c

```diff
@@ -186,6 +188,13 @@ NTSTATUS KSIAPI KphpInitializeProcessContext(
         {
             process->VerifiedProcess = TRUE;
         }
+
+        if (!KphpLsassIsKnown &&
```

Comment:
@ionescu007 what are the chances multiple lsass processes at once are important to handle?